package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cuckoofilter-native",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
I'm gonna make an actual tag and release for this package, to stop using a hash on metadata, as well as avoid weird behaviors in the node_modules if there's a conflict (two package at different git version but with the same version referenced in the package.json).

cc: @BourgoisMickael @jonathan-gramain